requirements-dev-frozen.txt

```diff
@@ -130,6 +130,7 @@ pytest-cov==4.0.0
 pytest-django==4.4.0
 pytest-rerunfailures==10.2
 pytest-sentry==0.1.11
+pytest-timeout==2.1.0
```

Comment:
damn, wish we didn't need a new dependency for this

Reply:
yeah, unfortuntately I don't think there is a better option

Reply:
there is another option in theory -- we could write a little teardown that fails the test without interrupting it -- that would eliminate the need for a dep and also it could avoid the retrying on timeout issue

but this is probably the easiest for now!